gtsfm/evaluation/merge_reports.py

```diff
@@ -0,0 +1,107 @@
+"""Merge two GTSFM reports for side-by-side comparison.
+
+Authors: John Lambert
+"""
+
+import argparse
+from collections import defaultdict
+from typing import Any, Dict, List, Tuple
+
+from tabulate import tabulate
+
+GTSFM_REPORT_TABLES = Dict[str, Dict[str, Any]]
+# contains info about each metric, from two separate reports
+GTSFM_MERGED_REPORT_TABLES = Dict[str, List[Tuple[str, Any, Any]]]
+
+
+def extract_tables_from_report(report_fpath: str) -> GTSFM_REPORT_TABLES:
```

Comment:
Have you looked into third-party libraries to do this?

Example: pandas does this out-of-the-box. https://pbpython.com/pandas-html-table.html. We can also merge dataframes pretty easily.